xml/cl.xml

```diff
@@ -255,6 +255,8 @@ server's OpenCL/api-docs repository.
         <type category="define">typedef <type>cl_bitfield</type>      <name>cl_platform_command_buffer_capabilities_khr</name>;</type>
         <type category="define">typedef <type>cl_bitfield</type>      <name>cl_mutable_dispatch_asserts_khr</name></type>
         <type category="define">typedef <type>cl_bitfield</type>      <name>cl_device_kernel_clock_capabilities_khr</name>;</type>
+        <type category="define">typedef <type>cl_ulong</type>      <name>cl_mem_device_address_EXT</name>;</type>
+        <type category="define">typedef struct _cl_mem_device_address_pair_EXT* <name>cl_mem_device_address_pair_EXT</name>;</type>
```

Comment:
This type is defined below as a `struct`, so an opaque handle definition is redundant and conflicting.
```suggestion
```